### PR TITLE
Parallel java

### PR DESCRIPTION
**Pull request by alan-stokes**
used for data out, fixes a interface change for x,y,p when only p is needed

needs 
- [ ] https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/307

### CODE REVIEW

File: spinnman/transceiver.py [hunk 1]
```diff
@@ -663,6 +663,8 @@ def _update_machine(self):
         # Get the coordinates of the boot chip
         version_info = self.get_scamp_version()
 
+        time.sleep(5)
```
Comment (dkfellows) [inline spinnman/transceiver.py]: Why _5_ seconds? Is this only needed with toroids? Would the big machine need longer?

**Comment by rowleya** [comment]
I *think* the need for magical sleep has now gone away from master, since I fixed it in #122

File: spinnman/transceiver.py (hunk 1 above)
Comment (rowleya) [reply spinnman/transceiver.py]: This is now literally the only change here, and I don't think it is needed.  I will therefore close this pull request.  Reopen if desired...

**Comment by alan-stokes** [comment]
i would hypothesis that the magic sleep was from data speed up, where that was the only way to fix the ethernet detection issue. since then, i think rowley's fixed it via other changes down stream. so the random sleep is redundant

**Pull request closed without merging**